src/main/java/data/TaskManager.java

```diff
@@ -181,15 +181,15 @@ public static void updateTask(LocalDate date, int taskIndex, String newTaskDescr
                 String deadlineResponse = scanner.nextLine().trim().toLowerCase();
                 if (deadlineResponse.equals("yes")) {
                     System.out.println("Enter the new deadline date and time, separated by a space:");
-                    String[] newDateAndTime = scanner.nextLine().trim().split(" ");
-                    task = new Deadline(newTaskDescription, newDateAndTime[0], newDateAndTime[1]);
+                    String[] newDatesAndTimes = scanner.nextLine().trim().split(" ");
+                    task = new Deadline(newTaskDescription, newDatesAndTimes[0], newDatesAndTimes[1]);
```

Comment:
Good use of coding standards, using plural variable names for collections or lists